lib/resources/r4/encounter.yaml

```diff
@@ -578,7 +592,7 @@ fields:
     <ul>
       <li>custom-attribute-name and custom-attribute-value are both required</li>
       <li>id and valueString for custom-attribute-name are required</li>
-      <li>custom-attribute-value can only be of type string</li>
+      <li>custom-attribute-value can be of type string or codeableConcept</li>
```

Comment:
I think we wanted to include a note about the supported systems for the codeableConcept

Reply:
https://github.com/cerner/fhir.cerner.com/pull/812/commits/a48d17ea0f5da9977dd29a235ff36d3034cea772#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR606

Reply:
not sure if this is the right way of adding these but let me know.

Reply:
Is there a way to show that the 71 from this system could be any value? "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/71"

Reply:
see if this is better https://github.com/cerner/fhir.cerner.com/pull/812/commits/cbc395d45dc3606a7fd1028eb1003ee011afd134#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR609

Reply:
https://github.com/cerner/fhir.cerner.com/pull/812/commits/97e5c761ca85b45b86998716653f32123d06d06c#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR609